app/templates/views/accessibility_statement.html

```diff
@@ -107,27 +107,27 @@ <h3 class="heading-small" id="non-compliance">Non-compliance with the WCAG versi
 
   <ol class="govuk-list govuk-list--number">
     <li>
-      The JAWS and NVDA screen readers do not announce the ‘copy to clipboard’ button. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">success criterion 4.1.3: status messages</a>. We plan to fix this in February 2023.
+      Some icons do not appear when using high contrast mode on Windows. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/visual-presentation.html">success criterion 1.4.8: visual presentation</a>. We have started working on this issue. We aim to fix it in March 2023.
```

Comment:
@tombye March is almost over - do we need to bump this?

```suggestion
      Some icons do not appear when using high contrast mode on Windows. This fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/visual-presentation.html">success criterion 1.4.8: visual presentation</a>. We have started working on this issue. We aim to fix it in March 2023.
```